include/swift/Option/FrontendOptions.td

```diff
@@ -287,6 +287,10 @@ def enable_experimental_flow_sensitive_concurrent_captures :
   Flag<["-"], "enable-experimental-flow-sensitive-concurrent-captures">,
   HelpText<"Enable flow-sensitive concurrent captures">;
 
+def enable_experimental_clang_importer_diagnostics :
+  Flag<["-"], "enable-experimental-clang-importer-diagnostics">,
+  HelpText<"Enable experimental ClangImporter diagnostics">;
```

Comment:
Maybe this help text could be better. What about, "Enable experimental diagnostics when importing C, C++, and Objective-C libraries."